### DISCUSSION TITLE
coordinate file in "sys_configs"

### DISCUSSION THREAD

**Discussion by cui-hq** [deepmodeling/dpgen#1104] [Q&A]
Is it just support POSCAR format, can I use pdb or xyz?

**Comment by njzjz** [comment]
See #441, #508, #993.

**Reply by ramonmi** [reply]
As mentioned in https://github.com/deepmodeling/dpgen/discussions/441, the `.xyz` format file lacks ABC info. I find that when I used `.xyz` format file in `sys_configs`, the ABC info of the `confs/000.0000.lmp` file in 01.model_devi step has been added with *100* by default, as shown in the figure below.

- Could I get the right ABC info in `.lmp` file by supplementing it in the second row of `.xyz` file? PS: Now the second row of `.xyz` file is blank.
- Or, I must use poscar format to achieve the ideal results?

- use `.xyz` in `sys_configs`
![image](https://github.com/user-attachments/assets/44d5f773-9012-4b36-9165-97469f78d64d)
- use poscar in `sys_configs`
![image](https://github.com/user-attachments/assets/4bca9c8e-3309-4bb1-b378-b5c6f2f2dd16)
